plugins/metadata/services/MetadataService.php

```diff
@@ -229,6 +232,7 @@ function addFromUrlAction($metadataProfileId, $objectType, $objectId, $url)
 	 */
 	function addFromBulkAction($metadataProfileId, $objectType, $objectId, $url)
 	{
+		myUploadUtils::isUrlFileTypeRestricted($url);
```

Comment:
the function already checks this in addFromUrlAction so dont need to check here as well